cache/in-memory/src/event/member.rs

```diff
@@ -180,6 +184,10 @@ impl UpdateCache for MemberRemove {
         if remove_user {
             cache.users.remove(&self.user.id);
         }
+
```

Comment:
Move this above the `RecourceType::MEMBER` check and check whether the user wants to cache guilds `ResourceType::GUILD`

Reply:
done